lib/topological_inventory/satellite/connection.rb

```diff
@@ -40,10 +61,11 @@ def receptor_controller_base_url
         "#{scheme}://#{host}:#{port}"
       end
 
+      # org_id with any number is required by receptor controller
       def identity_header(account_number)
         {
           "x-rh-identity" => Base64.strict_encode64(
-            {"identity" => {"account_number" => account_number}}.to_json
+            {"identity" => {"account_number" => account_number, "internal" => {"org_id" => '000002'}}}.to_json
```

Comment:
This is a little weird, do you know why this is?

Reply:
I'm waiting for response